README.md

```diff
@@ -1,25 +1,48 @@
-## Metrics API for erldns
+# Metrics API for [erldns](https://github.com/dnsimple/erldns)
 
 This app provides an HTTP API for gathering and querying metrics from an erldns server and presenting those metrics as JSON.
 
 Here's an example script that shows how to get the output with curl and pass through Python to format it in a pretty fashion.
 
-```sh
+```bash
 curl -s http://localhost:8082/ -H "Accept: application/json" | python -mjson.tool
 ```
 
 Note that timing stats are givin in microseconds.
 
-### Configuration
+## Configuration
 
 To configure the metrics API port, add something like the following to your erlang configuration section:
 
 ```erlang
-[ 
+[
   {erldns,[
       {metrics, [
         {port, 8082}
       ]},
     ]}
 ]
 ```
+
+## Building
+
+To build:
+
+```bash
+make
+```
+
+To start fresh:
+
+```bash
+make fresh
+make
```

Comment:
`make fresh` should clear the artifacts and execute the build, so the second `make` should not be necessary.